policies/aws_cloudtrail_policies/aws_cloudtrail_enabled.yml

```diff
@@ -73,3 +73,130 @@ Tests:
         "GlobalEventSelectors": null,
         "Trails": []
       }
+  - 
+    Name: Advanced Selector With Management
+    ExpectedResult: true
+    Resource:
+      {
+        "TrailARN": "arn:aws:cloudtrail:us-east-1:412345:trail/write_to_bucket",
```

Comment:
this should continue to be called `Trails: []` on the meta object

AWS account ids should always be 12 digits

Reply:
the trails key is unchanged, the only change is the addition of GlobalAdvancedEventSelectors field on Cloudtrail.Meta resources and AdvancedEventSelectors on Cloudtrail resources.

ex. Cloudtrail.Meta resource
```	
        "AccountId": "135923316767",
	"AdvancedEventSelectors": [
		{
			"FieldSelectors": [
				{
					"EndsWith": null,
					"Equals": [
						"Data"
					],
					"Field": "eventCategory",
					"NotEndsWith": null,
					"NotEquals": null,
					"NotStartsWith": null,
					"StartsWith": null
				},
				{
					"EndsWith": null,
					"Equals": [
						"AWS::S3::Object"
					],
					"Field": "resources.type",
					"NotEndsWith": null,
					"NotEquals": null,
					"NotStartsWith": null,
					"StartsWith": null
				}
			],
			"Name": null
		},
		{
			"FieldSelectors": [
				{
					"EndsWith": null,
					"Equals": [
						"Data"
					],
					"Field": "eventCategory",
					"NotEndsWith": null,
					"NotEquals": null,
					"NotStartsWith": null,
					"StartsWith": null
				},
				{
					"EndsWith": null,
					"Equals": [
						"AWS::CloudTrail::Channel"
					],
					"Field": "resources.type",
					"NotEndsWith": null,
					"NotEquals": null,
					"NotStartsWith": null,
					"StartsWith": null
				}
			],
			"Name": null
		}
	],
	"GlobalEventSelectors": [
		{
			"DataResources": null,
			"ExcludeManagementEventSources": null,
			"IncludeManagementEvents": true,
			"ReadWriteType": "All"
		}
	],
	"Name": "AWS.CloudTrail.Meta",
	"Region": "global",
	"ResourceId": "135923316767::AWS.CloudTrail.Meta",
	"ResourceType": "AWS.CloudTrail.Meta",
	"Trails": [
		"arn:aws:cloudtrail:eu-west-1:135923316767:trail/test-cloudtrail-events",
		"arn:aws:cloudtrail:us-west-2:292442345278:trail/PantherLabsOrganizationTrail"
	]
}
```

Reply:
this was meant as a reply to your other comment

>We should get with the connections team and see how this will be formatted as it ships, we can possibly pick up their test case(s) as our own. my understanding is that the Trails: key will persist, though I'm not 100% sure of how the AdvancedEventSelectors are going to manifest on the AWS.CloudTrail.Meta obje